vcfa/sample_vcfa_test_config.json

```diff
@@ -0,0 +1,67 @@
+{
+  "//": "Sample test configuration file for VCFA",
+  "//": "Rename this file to vcfa_test_config.json and change the values to suit your system",
+  "//": "You can safely remove the comments or leave them. The parser will ignore them.",
+  "provider": {
+    "//"  : "This section contains credentials related to the vCD connection of Sys or Org user",
+    "user": "root",
+    "password": "somePassword",
+    "tmVersion": "10.7.0",
+    "tmApiVersion": "40.0",
+    "token": "Access token to be used instead of username/password",
+
+    "//": "If useSamlAdfs is true - client will try to authenticate against ADFS using SAML.",
+    "useSamlAdfs": false,
+    "//": "customAdfsRptId allows to specify custom Relaying Party Trust Identifier. By default",
+    "//": "client will use vCD SAML Entity ID",
```

Comment:
Not sure we should leave VCD mention here, as we're connecting to TM in this context. This question will likely popup in other situations like here too.